src/utils/framework.ts

```diff
@@ -75,6 +79,20 @@ interface LevelOutput {
     nodes?: LevelOutput[];
 }
 
+interface WidgetData {
+    data: {
+        value: string;
```

Comment:
Is value always string for WidgetData?

Reply:
I made the interface based on the incoming data. I don't exactly know whether all values are string or other values are possible too.

Reply:
Maybe it differs from widget to widget